src/main/java/codeu/model/data/User.java

```diff
@@ -130,8 +128,7 @@ public UserBuilder bio(String newBio) {
 	}
 	
 	public User createUser() {
-	    return new User(nestedId, nestedName, nestedPassword, nestedCreation, nestedAge, nestedFirstName, 
-		    nestedLastName, nestedEmail, nestedPhoneNum, nestedBio);
+	    return new User(nestedId, nestedName, nestedPassword, nestedCreation);
```

Comment:
You should make a private constructor that has all of these parameters. Right now, even if you populate all the information, when you call createUser, everything will be set to defaults since you call the 4 parameter constructor. Sorry, I should clarify that it's okay to have **private** constructors with lots of parameters.

That is, add: private User(UUID id, String name, String password, Instant creation, Integer age, .........)
and have createUser call this new constructor.